app/controllers/projects/settings/project_custom_fields_controller.rb

```diff
@@ -106,4 +106,11 @@ def bulk_edit_service
         project_custom_field_section: @project_custom_field_section
       )
   end
+
+  def breadcrumb_items
+    [{ href: project_overview_path(@project.id), text: @project.name },
+     { href: project_settings_general_path(@project.id), text: I18n.t("label_project_settings") },
+     t("settings.project_attributes.heading")]
+  end
+  helper_method :breadcrumb_items
```

Comment:
It would be cleaner if those methods wouldn't be in the controller but rather in a component. For some of the pages no dedicated header component exists yet, so they will need to be created in those cases. Alternatively, the code should be placed inside the view or in the corresponding helper.